doc/whats-new.rst

```diff
@@ -24,14 +24,18 @@ Breaking changes
 - Remove support for Python 2. This is the first version of xarray that is
   Python 3 only. (:issue:`1876`).
   By `Joe Hamman <https://github.com/jhamman>`_.
-- The `compat` argument to `Dataset` and the `encoding` argument to 
+- The `compat` argument to `Dataset` and the `encoding` argument to
   `DataArray` are deprecated and will be removed in a future release.
   (:issue:`1188`)
   By `Maximilian Roos <https://github.com/max-sixty>`_.
 
 Enhancements
 ~~~~~~~~~~~~
 
+- Internal plotting now supports `cftime.datetime` objects as time series.
+  (:issue:``2164``)
+  By `Spencer Clark <https://github.com/spencerkclark>` and
```

Comment:
I appreciate the shout-out for the upstream change, but please list your name first here, as this is your PR in xarray :)